lib/node_modules/@stdlib/math/base/special/log1mexp/README.md

```diff
@@ -77,6 +77,97 @@ for ( i = 0; i < x.length; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/log1mexp.h"
+```
+
+#### stdlib_base_log1mexp( x )
+
+Evaluates the [natural logarithm][@stdlib/math/base/special/ln] of `1-exp(-|x|)`.
+
+```c
+double out = stdlib_base_log1mexp( 0.0 );
+// returns -INFINITY
```

Comment:
I'd choose a slightly different example here, as it is not guaranteed that all compilers will return the same value for "infinity". E.g., use `0.1` instead of `0.0`.

Reply:
Okay sure!